tracer/src/Datadog.Trace/Configuration/Schema/DatabaseSchema.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="DatabaseSchema.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System.Collections.Generic;
+
+namespace Datadog.Trace.Configuration.Schema
+{
+    internal class DatabaseSchema
+    {
+        private readonly SchemaVersion _version;
+        private readonly string _defaultServiceName;
+        private readonly IDictionary<string, string>? _serviceNameMappings;
+
+        public DatabaseSchema(SchemaVersion version, string defaultServiceName, IDictionary<string, string>? serviceNameMappings)
+        {
+            _version = version;
+            _defaultServiceName = defaultServiceName;
+            _serviceNameMappings = serviceNameMappings;
+        }
+
+        public string GetOperationName(string databaseType) => $"{databaseType}.query";
```

Comment:
I find it amusing that we (APM) been trying to move from "operation name" to "span name" for years but somehow it won't go away 😅

Reply:
Probably for a follow-up PR with performance optimizations, but should we try caching this result for each `databaseType` to avoid allocating the new string every time? Maybe only cache a single value if `databaseType` rarely changes within the service's lifetime?

Reply:
Thanks for keeping performance in mind :+1: We do still have the type `Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.DbScopeFactory.Cache<TCommand>` which caches the service name based on the AdoNet implementation. We could probably re-use that work to accomplish the caching

Reply:
I'll leave this work for a separate PR though